extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpBuildTimeConfig.java

```diff
@@ -35,4 +35,13 @@
      */
     @ConfigItem(defaultValue = "/quarkus")
     public String consolePath;
+
+    /**
+     * The HTTP root path for framework endpoints. Various endpoints such as metrics, health,
+     * and open api are deployed under this path.
+     * Setting the value to "/" disables the separate framework root,
+     * resulting in all framework endpoints being served from "/" along with the application.
+     */
+    @ConfigItem(defaultValue = "/q")
+    public String frameworkRootPath;
```

Comment:
So, we have /quarkus for the console and /q for framework.
I may be wrong but I don't believe the console should be "application". 

Also, all guides, workshops and quickstarts would need an update once this is merged

Reply:
Is that the new dev console that was created as part of Reactive Resteasy? Did that get merged already, might have missed it.

If it's in, would /q/console be ok?

Reply:
I let @FroMage @stuartwdouglas and @geoand decide.

Reply:
The console started out as part of RESTEasy Reactive, but is now in a separate (not yet merged) branch. The console currently is under `/@dev`.

Reply:
Who decided on the `/q` default prefix? Isn't it likely to conflict with user paths?

Reply:
It was agreed a few weeks ago in a team meeting, but not set in stone

Reply:
I guess I would have made it `/@q` to make sure it didn't conflict. But if everyone prefers `/q` let's go with that.

As for the dev console it should move to `/q/dev` if we do that.

Reply:
I think `@` is reserved for a special meaning within a scheme (eg. https://foo:bar@example.com)

Reply:
But looking at the [URL RFC](https://tools.ietf.org/html/rfc1738) I see that `@` is accepted, so I dunno 😃

Reply:
It's only reserved as part of the `authority` section (host name).